src/server/routes/fileProcessing.js

```diff
@@ -29,7 +31,7 @@ router.post('/:meter_id', upload.single('csvFile'), async (req, res) => {
 			}
 		}
 	};
-	if (!validate(req.params, validParams).valid || !req.file.buffer) {
+	if (!validate(req.params, validParams).valid) {
```

Comment:
Why was `!req.file.buffer` removed?

Reply:
This sometimes creates problem if users try to upload wrong file format (not csv). Since buffer is undefined at that point. So, this was removed.